layouts/partials/footer.html

```diff
@@ -1,22 +1,395 @@
+      <li>
+        <a href="https://notaryproject.dev/docs/how-to/">How-to guides</a>
+      </li>
+      <li><a href="https://notaryproject.dev/docs/concepts/">Concepts</a></li>
+      <li>
+        <a href="https://notaryproject.dev/docs/troubleshooting/"
+          >Troubleshooting</a
+        >
+      </li>
+      <li><a href="https://notaryproject.dev/docs/faq/">FAQs</a></li>
+    </ul>
   </div>
-  <div class="row text-center text-white small">
-    <div class="col-12 text-center py-2 order-sm-2">
-      <a href="https://www.linuxfoundation.org/terms" target="_blank" rel="noopener">Terms</a> |
-      <a href="https://www.linuxfoundation.org/privacy" target="_blank" rel="noopener">Privacy</a> |
-      <a href="https://www.linuxfoundation.org/trademark-usage" target="_blank" rel="noopener">Trademarks</a> |
-      <a href="{{ .Site.Params.github_repo }}/blob/{{ .Site.Params.github_branch }}/LICENSE" target="_blank" rel="noopener">License</a>
-    </div>
+
+  <div class="footer-list-blog">
+    <h4>Blogs</h4>
+    <ul>
+      <li><a href="https://notaryproject.dev/blog/">Posts in 2023</a></li>
+      <li><a href="https://notaryproject.dev/blog/">Posts in 2022</a></li>
+      <li><a href="https://notaryproject.dev/blog/">Posts in 2021</a></li>
+    </ul>
+  </div>
+
+  <div class="footer-list-com">
+    <h4>Communities</h4>
+    <ul>
+      <li>
+        <a href="https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/">Slack</a>
+      </li>
+      <li>
+        <a href="https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw"
+          >Notary Project Meeting doc</a
+        >
+      </li>
+      <li>
+        <a href="https://hackmd.io/wii3-L8ZQZ-U3ET0XNY8Gg">TUF Meeting doc</a>
+      </li>
+      <li>
+        <a
+          href="https://www.youtube.com/playlist?list=PL1ykZdgmLkb7SlXax-hJVUgvNHmq4Cyz9"
+          >YouTube</a
+        >
+      </li>
+    </ul>
   </div>
-  <div>
-    <small class="text-white">
-      &copy; copyright {{ now.Year }} | All Rights Reserved
-    </small>
+
+  <div class="f-line"></div>
+
+  <div class="f-l-block">
+    <div class="f-legals">
+      <div class="f-logo">
+        <a href="{{ .Site.Home.RelPermalink }}">
+          {{ $image := resources.Get "icons/logo-dark.png" }}
+          <img
+            src="{{ $image.RelPermalink }}"
+            alt="notary logo"
+            class="f-logo"
+          />
+        </a>
+      </div>
+
+      <p class="f-cr">© copyright 2023 | All Rights Reserved</p>
+    </div>
+
+    <div class="f-terms">
+      <div class="f-t-links">
+        <a
+          href="https://www.linuxfoundation.org/terms"
+          target="_blank"
+          rel="noopener"
+          class="f-text"
+          >Terms</a
+        >
+        <a
+          href="https://www.linuxfoundation.org/privacy"
+          target="_blank"
+          rel="noopener"
+          class="f-text"
+          >Privacy</a
+        >
+        <a
+          href="https://www.linuxfoundation.org/trademark-usage"
```

Comment:
These four links should be placed in one line. This is the preview on my 14' inch screen. Please also add a YouTube icon.
![image](https://github.com/notaryproject/notaryproject.dev/assets/40452856/357013b4-5650-4467-ad82-a3b538df32ed)